examples/labels-2d.py

```diff
@@ -4,17 +4,22 @@
 """
 
 from skimage import data
+from skimage.color import rgb2gray
 from skimage.segmentation import slic
 from napari import ViewerApp
 from napari.util import app_context
 
 
 with app_context():
     astro = data.astronaut()
+
     # initialise viewer with astro image
-    viewer = ViewerApp(astronaut=astro, multichannel=True)
+    viewer = ViewerApp(astronaut=rgb2gray(astro), multichannel=False)
+    viewer.layers[0].colormap = 'gray'
+
     # add the labels
     # we add 1 because SLIC returns labels from 0, which we consider background
     labels = slic(astro, multichannel=True, compactness=20) + 1
-    label_layer = viewer.add_labels(labels, name='SLIC segmentation')
+    labels[:50,:50] = 0
```

Comment:
I understand that this is nice for debugging, but it also makes no sense... How about adding an example, `labels-0-2d.py`, where we use thresholding + ndi.label for segmentation, thus obtaining a background label? Maybe something with coins(), or cell nuclei?

Reply:
I have added a new example with coins based on - http://scikit-image.org/docs/dev/auto_examples/segmentation/plot_label.html

I think it looks quite nice